config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  root 'welcome#index'
+
+  namespace :admin do
```

Comment:
can we also add a route for /admin which is either the admin index (can just be a hello world page for now) if there is an admin session logged in, or redirects to admin/login otherwise?

Reply:
I've added in a before filter for this and an index action for an admin session